File: camp/week4/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int arr[3]= {0,0,0};
        for(auto i : nums){
            arr[i]++;
        }
        int s = 0;
        for(int i = 0; i<3; i++){
            while(arr[i] && s<nums.size()){
                nums[s]=i;
                arr[i]--;
                s++;
            }
        }
    }
};

File: camp/week6/LeetCode/SortColors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        i=0
        for k in range(len(nums)):
            if nums[k] == 0:
                nums[k],nums[i] = nums[i], nums[k]
                i+=1
        for k in range(i,len(nums)):
            if nums[k] == 1:
                nums[k],nums[i] = nums[i], nums[k]
                i+=1